package PBMessages;

message NextPrimeResult
{
	required uint64 taskId = 1;
	required uint32 nextPrime = 2;
}
